src/frontend/src/app/assets/assets.component.ts

```diff
@@ -7,14 +7,35 @@ import { AssetsService, AlertService } from '../services/index';
   styleUrls: ['./assets.component.css']
 })
 export class AssetsComponent implements OnInit {
+  public limit: Number = 0;
+  public offset: Number = 0;
   assets = [];
```

Comment:
on first page, seems its limit = 20 and offset =0; also see getAssetReadingS method